tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -259,10 +259,10 @@ It is possible to define functions in a Godot shader. They use the following syn
     }
 
 
-You can only use functions that are defined above the function you are calling 
-them from.
+You can only use functions that have been defined above (higher in the editor) the function from which you are calling 
+them.
 
-Function argument can have special qualifiers:
+The function argument can have special qualifiers:
```

Comment:
This should probably read "Function arguments ..." because we are mentioning functions in general, not a specific function.